From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sun, 3 Jan 2021 12:21:40 +0000
Subject: spelling

---
 test_more/general/adfun.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test_more/general/adfun.cpp b/test_more/general/adfun.cpp
index efa8764..59a593c 100644
--- a/test_more/general/adfun.cpp
+++ b/test_more/general/adfun.cpp
@@ -24,7 +24,7 @@ bool adfun_empty(void)
 {   bool ok = true;
     size_t thread  = CppAD::thread_alloc::thread_num();
     //
-    // Allocate this memory first incase we are using
+    // Allocate this memory first in case we are using
     // CppAD::vector which uses thread_alloc.
     CPPAD_TESTVECTOR( CppAD::AD<double> ) ax(1), ay(1);
     //
