From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Thu, 10 Apr 2014 10:31:13 +0100
Subject: autotools warnings

---
 configure.ac | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configure.ac b/configure.ac
index bde91b6..c29e9bc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -481,6 +481,9 @@ AM_INIT_AUTOMAKE([nostdinc subdir-objects -Wall])
 dnl Use LIBTOOL for cppad_ipopt ?
 dnl AC_PROG_LIBTOOL
 
+dnl required by automake when linking libraries using a non-POSIX linker
+AM_PROG_AR
+
 dnl Determine if ranlib is present and where it is if present (set RANLIB)
 AC_PROG_RANLIB
 
